lib/manager/components/HomeProjectDropdown.js

```diff
@@ -15,88 +15,50 @@ type Props = {
+          ? <LinkContainer className='pull-right' style={{position: 'relative', listStyle: 'none', zIndex: '1', marginBottom: '5px'}} to={`/project/${selectedOption && selectedOption.value}`}>
+            <Button bsStyle='primary'>View {selectedOption && selectedOption.label}</Button>
           </LinkContainer>
           : null
         }
-        <DropdownButton
-          id='context-dropdown'
-          title={activeProject
-            ? <span>
-              <Icon type='folder-open-o' /> {abbreviatedProjectName}
-            </span>
-            : <span>
-              <img alt={profile.email} height={20} width={20} src={profile.picture} />
-              {' '}
-              {profile.nickname}
-            </span>
-          }
-        >
-          {activeProject && (
-            <LinkContainer key='home-link' to={`/home/`}>
-              <MenuItem>
-                <span>
-                  <img alt={profile.email} height={20} width={20} src={profile.picture} />
-                  {' '}
-                  {profile.nickname}
-                </span>
-              </MenuItem>
-            </LinkContainer>
-          )}
-          {activeProject && <MenuItem key='divider' divider />}
-          {visibleProjects.length > 0
-            ? visibleProjects.map((project, index) => {
-              if (activeProject && project.id === activeProject.id) {
-                return null
-              }
-              return (
-                <LinkContainer to={`/home/${project.id}`} key={project.id}>
-                  <MenuItem eventKey={project.id}>
-                    <Icon type='folder-o' /> {project.name}
-                  </MenuItem>
-                </LinkContainer>
-              )
-            })
-            : null
-          }
-          {(activeProject && visibleProjects.length > 1) || !activeProject ? <MenuItem divider /> : null}
-          {
-            isAdmin && (
-              <LinkContainer to='/settings/organizations'>
-                <MenuItem>
-                  <Icon type='users' /> Manage projects
-                </MenuItem>
-              </LinkContainer>
-            )
-          }
-          {
-            isAdmin && <MenuItem divider />
-          }
-          {
-            isAdmin && (
-              <LinkContainer to='/project/new'>
-                <MenuItem>
-                  <Icon type='plus' /> Create project
-                </MenuItem>
-              </LinkContainer>
-            )
-          }
-        </DropdownButton>
+        <Select
+          value={selectedOption}
+          style={{marginBottom: '5px', zIndex: '0'}}
+          onChange={this.handleChange}
+          options={visibleProjects.map(p => ({value: p.id, label: p.name}))} placeholder={`Select Project...`}
+        />
+        <LinkContainer to='/project/new' style={{listStyle: 'none'}}>
+          <MenuItem>
+            <Icon type='plus' /> New project
+          </MenuItem>
+        </LinkContainer>
```

Comment:
This should remain only accessible to admins (see the check that was removed for `isAdmin`).